apps/aecore/include/aec_db.hrl

```diff
@@ -5,7 +5,7 @@
 -record(aec_call_state         , {key, value}).
 -record(aec_contract_state     , {key, value}).
 -record(aec_chain_state        , {key, value}).
--record(aec_block_state        , {key, value, difficulty, fork_id, fees, key_hash}).
+-record(aec_block_state        , {key, value, difficulty, fork_id, fees, key_hash, fraud}).
```

Comment:
Perhaps we may want to introduce separate storage to indicate fraud key blocks, so that aec_block_state remains lightweight? 🤔